packages/itwin/map-layers/vitest.config.mts

```diff
@@ -0,0 +1,28 @@
+import { coverageConfigDefaults, defineConfig } from "vitest/config";
+
+export default defineConfig({
+  test: {
+    globals: true,
+    environment: "jsdom",
+    dir: "src",
+    setupFiles: ["src/test/setup.ts"],
+    coverage: {
+      provider: "v8",
+      include: [
+        "src/**/*"
+      ],
+      exclude: [
+        ...coverageConfigDefaults.exclude,
+        "src/test/**/*",
+        "**/*.d.ts",
+        "**/*.d.tsx"
+      ],
+      reporter: [
+        "text-summary",
+        "lcov",
+        "cobertura"
+      ],
+      reportsDirectory: "./lib/cjs/test/coverage",
+    }
```

Comment:
can you add [minWorkers](https://vitest.dev/config/#minworkers) and maxWorkers to the config? Look at the vitest configs in core, they can match those numbers. We just want to avoid resource starvation